include/cutlass/gemm/device/gemm_universal_base.h

```diff
@@ -262,7 +276,7 @@ class GemmUniversalBase {
           return -1;
         }
 
-        smem_capacity = static_cast<int>(properties.sharedMemPerMultiprocessor);
+        smem_capacity = static_cast<int>(properties.sharedMemPerBlockOptin);
```

Comment:
thank you for the change.  I think you would need a rebase with the latest 2.11.  this line is already changed in tot.

Reply:
Thanks! I rebased both versions and also updated the device identifier, which was undetermined in the last commits